TF_Settings_Web/src/Components/TFButton/TFButtons.tsx

```diff
@@ -28,6 +27,32 @@ interface TextButtonProps extends BaseButtonProps {
     textStyle?: React.CSSProperties;
 }
 
+const getButtonTextElements = (
```

Comment:
💬 I'm really not a fan of the "function that returns JSX" pattern where something could be a Function Component itself. Could this be changed so it's usage would be something like `<ButtonText title='' titleStyle={...} />

Reply:
Updated